examples/typescript-nodejs/README.md

```diff
@@ -0,0 +1,50 @@
+Example Node.js project consuming TypeScript types
+==================================================
+
+This Node.js project consumes the [Digital Identity Vocab](https://github.com/govuk-one-login/data-vocab) TypeScript types.
+
+In the `package.json`, note the `@govuk-one-login/data-vocab` dependency:
+
+```json
+"@govuk-one-login/data-vocab": "1.4.2"
+```
+
+> **Note**
+> See [releases](https://github.com/govuk-one-login/data-vocab/releases) for the latest version.
+
+### Build the project
+
+##### Prerequisities
+
+This project uses TypeScript and expects the `tsc` command to be available.
```

Comment:
So I can see that once I've npm-installed `tsc` is available by running `./node_modules/typescript/bin/tsc` ...

Is it normal to also install it globally? Can I get confusing effects if I have a different global tsc to the one I've installed locally?

I could run the npm build / start scripts without installing tsc globally, anyway.

Reply:
Oh, that's perfect - I'll update the docs :)